### PR TITLE
Add break timer

### PR DESCRIPTION
**Pull request by dbartholomae**
I've written a first implementation based on this proposal I added in #119. It would be helpful for me to have on Monday, but if you need more time, I will be able to live without it a bit longer ;)
I had to make some additional smaller clean ups around the the Timer subscription and usage of currentTime from state to be able to use the Timer subscription for multiple timers.

### CODE REVIEW

**Comment by dbartholomae** [comment]
By the way, if it helps to reduce the effort on your end, I'm happy to have a call and discuss the changes :)

**Comment by mrozbarry** [comment]
I'm really on the fence with this feature. The code looks good, so I'm not concerned with that, but I'd want feedback on how mobs currently manage their breaks. For reference, I have at least 10 teams at work that manage breaks on their own while still using mobtime, so I'm curious what the value add is.

**Comment by mrozbarry** [comment]
Also, sorry it took me a while to get back to you on this!

**Comment by dbartholomae** [comment]
I'm currently running two mobs with this version of the timer (hosted at https://mobtime.herokuapp.com/ ). I know that others asked me for that version as well, but I didn't yet get feedback from them whether they actually use it.
The main reasons for us to use the integrated break timers are:
1. Having only one tool to set up and look at as a mob instead of having to sync on multiple tools.
2. Being reminded of breaks less intrusively as the timer just stops you from starting the next round, but doesn't interrupt your current one (which often lead us to "first finish what we are up to" and then forget the break all together)

**Comment by adamkasztenny** [comment]
Our mob would really appreciate this feature as well. Our concerns are the same as @dbartholomae 's; managing two timers is clunky (especially when its not shared), and being forced to take breaks is always nice.

File: public/actions.break.test.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+  t.is(state.breakTimerStartedAt, initialState.breakTimerStartedAt);
+  t.is(effect, undefined);
+});
+
+test('does not start a break timer if one is already running', t => {
+  const initialState = {
+    breakTimerStartedAt: new Date(),
+    settings: {
+      breaksEnabled: false,
```
Comment (adamkasztenny) [inline public/actions.break.test.js:65]: Shouldn't this be `true`?

Comment (dbartholomae) [reply public/actions.break.test.js]: Good point, fixed

**Comment by mrozbarry** [comment]
So a lot has changed in the code-base since this PR. `master` is the base again (dev will probably be deleted in the near future), and on top of that, I am thinking of building a bot for break timers rather than a direct integration. With that in mind, I'm going to close this, but maybe there will be an opportunity in the future to revisit this.

**Pull request closed without merging**